framework/meta/src/cli_args/cli_args_standalone.rs

```diff
@@ -76,6 +79,21 @@ pub struct InfoArgs {
     pub ignore: Vec<String>,
 }
 
+
+
+#[derive(Default, Clone, PartialEq, Eq, Debug, Args)]
+pub struct TestArgs {
+    /// Target directory where to generate contract integration tests.
+    /// Will be current directory if not specified.
+    #[arg(short, long, verbatim_doc_comment)]
+    pub path: Option<String>,
+
+    /// This arg can differentiate between types of tests to be ran (go, scenario, all).
+    /// Default value will be "rust" if not specified.
+    #[arg(short, long, default_value = "rust", verbatim_doc_comment)]
+    pub test_type: String,
```

Comment:
I would use a `--go` argument here, and maybe separately `--scen` for running the scenario tool.

Reply:
Good idea, it's cleaner this way